### PR TITLE
Spring k8s cassandra blog

### PR DESCRIPTION
**Pull request by BrianMMcClain**
@bryanfriedman @tybritten Curious on your thoughts on how I've called out that this is a guest blog since we don't have a team member to link to. I've also cleaned up the featured pieces a bit.

### CODE REVIEW

File: content/topics/kubernetes/_index.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Kubernetes
 weight: 2
+featured: true
```
Comment (bryanfriedman) [inline content/topics/kubernetes/_index.md:4]: Seems weird to highlight a topic page. Maybe we highlight the most recent other blog as well?

File: content/blog/spring-k8s-cassandra.md [hunk 2]
```diff
@@ -0,0 +1,451 @@
...
+- Spring Cloud
+- Kubernetes
+patterns:
+- API
+# Author(s)
+team: 
+- Chris Splinter
+---
+
+> This blog was written by Chris Splinter who works for DataStax, focusing on open source solutions in the Apache Cassandra ecosystem. As a former automation engineer, Chris is focused on developers and making it easier to build applications through APIs, frameworks, and tools. You can find him on [LinkedIn](https://www.linkedin.com/in/csplinter/).
```
Comment (bryanfriedman) [inline content/blog/spring-k8s-cassandra.md:27]: I think this works! I wonder about putting it on the bottom instead even though it is a long post.

File: content/topics/kubernetes/_index.md (hunk 1 above)
Comment (BrianMMcClain) [reply content/topics/kubernetes/_index.md:4]: Sure thing, let me update it with a new commit

File: content/blog/spring-k8s-cassandra.md (hunk 2 above)
Comment (BrianMMcClain) [reply content/blog/spring-k8s-cassandra.md:27]: That works too, I was debating if it'd be better at the top or bottom so I'm not too picky

**Comment by BrianMMcClain** [comment]
@bryanfriedman Alright, changed around the two things you mentioned

**Pull request merged**